app/forms/steps/outgoings/council_tax_form.rb

```diff
@@ -0,0 +1,40 @@
+module Steps
+  module Outgoings
+    class CouncilTaxForm < Steps::BaseFormObject
+      include Steps::HasOneAssociation
+      has_one_association :outgoings
+
+      attribute :pays_council_tax, :value_object, source: YesNoAnswer
+      attribute :council_tax_amount, :integer
```

Comment:
One thing here is that by having it as an integer we lose pence amounts. Not sure how much of an issue this is but you can input pounds and pence and submit and then when you come back the pence amount after the decimal point is lost.

Reply:
👍 Changed to string here, and in db table. Will convert to pence integer in the serialization before sending to datastore, and back again on rehydration

Reply:
Storing now as council_tax_amount in pence